tests/fixtures.py

```diff
@@ -10,13 +8,18 @@
 @fixture(name="dvc_repo_session", scope="session")
 def fix_dvc_repo_session() -> DvcRepo:
     with tempfile.TemporaryDirectory() as dir:
-        # os.mkdir("./temp_test")
         subprocess.check_call(["git", "init"], cwd=dir)
+        print(dir)
         dvc = DvcRepo.init(dir, subdir=True)
         yield dvc
         dvc.close()
 
 
 @fixture(name="dvc_repo")
-def fix_dvc_repo() -> DvcRepo:
-    return fix_dvc_repo_session()
+def fix_dvc_repo(dvc_repo_session) -> DvcRepo:
+    subprocess.check_call(
+        f"cp -r {dvc_repo_session.root_dir} {dvc_repo_session.root_dir}_1",
+        shell=True,
+    )
```

Comment:
1. Rather than add a suffix, we want to create a new temp dir -- so the directory will be cleaned up automatically.
2. need to yield the result (inside of `with` for temp directory creation).
3. Although its good that you've got docker working, eventually we want to have windows compatibility -- shutil.copytree is better as it doesn't depend on os -- "cp" (perhaps you had a problem with it?)

Reply:
I have modified the code in my coming commit to use the same tempdir that the session fixture uses (and both have subdirectories in the tempdir) so that both are cleaned up at the same time. Let me know if this isn't something you want to do and still want to use a second tempdir.

cp was actually added to powershell as an alias, and now works in Windows as well. But I do agree that using Python code is a better idea in general. I will go ahead and make that change

Reply:
> I have modified the code in my coming commit to use the same tempdir that the session fixture uses (and both have subdirectories in the tempdir) so that both are cleaned up at the same time. Let me know if this isn't something you want to do and still want to use a second tempdir.

So the session has a fixed subdir, and the function-scoped fixture copies from that? That could work. Keep in mind that tests could run in parallel, depending on how pytest is run. So you'll want to choose unique names -- e.g. create a hash suffix for the directory. Seems like more work than just creating a new tempdir, but either way is ok, I guess.

Reply:
I see. That makes a lot of sense. Yeah I agree that tempdir is probably just better